packages/core/src/constants/type/Config.ts

```diff
@@ -88,6 +88,10 @@ export type FullConfig = {
    * Refresh standard calls each time the n-th block is mined.
    */
   refresh?: number | 'never' | 'everyBlock'
+  /**
+   * Optional Local storage override for use in environments like React Native
+   */
+  localStorageOverride: WindowLocalStorage['localStorage']
```

Comment:
```suggestion
  localStorageOverride?: WindowLocalStorage['localStorage']
```

The average user should not have to set it, or know that this exists.